sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -796,11 +796,11 @@ object SQLConf {
     .doc("Sets the compression codec used when writing ORC files. If either `compression` or " +
       "`orc.compress` is specified in the table-specific options/properties, the precedence " +
       "would be `compression`, `orc.compress`, `spark.sql.orc.compression.codec`." +
-      "Acceptable values include: none, uncompressed, snappy, zlib, lzo.")
+      "Acceptable values include: none, uncompressed, snappy, zlib, lzo, zstd.")
     .version("2.3.0")
     .stringConf
     .transform(_.toLowerCase(Locale.ROOT))
-    .checkValues(Set("none", "uncompressed", "snappy", "zlib", "lzo"))
+    .checkValues(Set("none", "uncompressed", "snappy", "zlib", "lzo", "zstd"))
```

Comment:
Btw, we cannot use `lz4` for the orc format?

Reply:
Is it supported before, @maropu ?

Reply:
I'm not familiar with this part though, I saw the ORC code: 
https://github.com/apache/orc/blob/f6b6b2ea70f1b74e2ffd897e8985ffcd1c082582/java/core/src/java/org/apache/orc/CompressionKind.java#L25-L27

Reply:
ORC does, but what I mean is that in Apache Spark.

Reply:
Ah, ok. nvm. Just a question.

Reply:
I just didn't use it before. Feel free to make a PR if you want to add it, @maropu .